example/ipsec_crypto/odp_ipsec_stream.c

```diff
@@ -65,6 +65,14 @@ void init_stream_db(void)
 	memset(stream_db, 0, sizeof(*stream_db));
 }
 
+void free_stream_db_intf(void)
+{
+	for (int i = 0; i < MAX_DB; i++) {
```

Comment:
Here you could just loop through strream_db->list to only go through the entries that actually exist.